connectors/AnthropicConnector/src/main/java/de/l3s/interweb/connector/anthropic/AnthropicConnector.java

```diff
@@ -0,0 +1,84 @@
+package de.l3s.interweb.connector.anthropic;
+
+import java.time.Instant;
+import java.util.Map;
+import java.util.List;
+import java.util.ArrayList;
+
+import jakarta.enterprise.context.Dependent;
+
+import io.smallrye.mutiny.Uni;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import de.l3s.interweb.connector.anthropic.entity.AnthropicContent;
+import de.l3s.interweb.connector.anthropic.entity.CompletionBody;
+import de.l3s.interweb.core.ConnectorException;
+import de.l3s.interweb.core.completion.CompletionConnector;
+import de.l3s.interweb.core.completion.CompletionQuery;
+import de.l3s.interweb.core.completion.CompletionResults;
+import de.l3s.interweb.core.completion.Message;
+import de.l3s.interweb.core.completion.UsagePrice;
+import de.l3s.interweb.core.completion.Usage;
+import de.l3s.interweb.core.completion.Choice;
+
+@Dependent
+public class AnthropicConnector implements CompletionConnector {
+
+    private static final String version = "2023-06-01";
+    private static final Map<String, UsagePrice> models = Map.of(
+        "claude-3-opus-20240229", new UsagePrice(0.015, 0.075),
+        "claude-3-sonnet-20240229", new UsagePrice(0.003, 0.015),
+        "claude-3-haiku-20240307", new UsagePrice(0.00025, 0.00125)
+    );
+
+    @Override
+    public String getName() {
+        return "Anthropic";
+    }
+
+    @Override
+    public String getBaseUrl() {
+        return "https://api.anthropic.com/";
```

Comment:
This is reference url for displaying as a source. Should be `https://anthropic.com/`